src/app/modules/main/chat_section/module.nim

```diff
@@ -736,11 +736,16 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
 
   # Prepare notification
   let myPK = singletonInstance.userProfile.getPubKey()
+  if myPK == message.from:
+    # Any type of message coming from ourselves should never be shown as notification
+    return
```

Comment:
This condition can be moved above the Badge notif too. I don't think showing a badge for our own messages make sense either

Reply:
Good point! 👍